intellij-ide.md

```diff
@@ -0,0 +1,162 @@
+---
+layout: page
+title: Developing apps in the IntelliJ IDE
+sidebar: home_sidebar
+permalink: /intellij-ide/
+---
+
+Our custom [IntelliJ IDEA](https://www.jetbrains.com/idea/) plugins provides a
+fully integrated development experience in the IntelliJ IDE.
+
+* TOC Placeholder
+{:toc}
+
+## Installation and setup
+
+Please follow the [Flutter setup](/setup/) instructions to install IntelliJ and
+the Dart and Flutter plugins.
+
+### Updating the plugins<a name="updating"/>
+
+Updates to the plugins will be shipped on a regular basis. You should get
```

Comment:
No; usually plugins are updated infrequently, every 4 months as a new roll of IJ happens. We want to set the expectation that our plugin will be updated much, much more frequently than that.

Once we get a bit more mature we will slow down, and can remove this.